restapi.py

```diff
@@ -77,16 +78,15 @@ def on_cancel_help_request(self, request):
             return MethodNotAllowed()
 
         if request.method == "POST":
-            log.debug("Got cancel request: `%s`", request.form)
-            if False:
-                # TODO verify if the payload has all the required data
-                # TODO define parameters in the data
-                # do some processing, if there are any issues, return an error
-                return BadRequest("Request malformed")
+            try:
```

Comment:
Can you update the readme and add an example of such a request payload, so I know how to deal with it in the bot itself?

Reply:
Add it here: https://github.com/code4moldova/covid19md-voluntari-tg-client#payloads

Reply:
Done.